joomla/joomla-cms
# Cassiopeia Privacy Consent - Display Options Inline

Pull Request for Issue #37645 .

### Summary of Changes
The Privacy Consent decision is placed inline. It saves space and has a better usability 

### Testing Instructions
Turn on the Privacy consent plugin 
Turn on the Terms of Service plugin
Apply Patch an see if the radio buttons on the registration page and on the profile edit page display correct


### Actual result BEFORE applying this Pull Request
<img width="363" alt="grafik" src="https://user-images.githubusercontent.com/828371/166142356-2909f6cc-b41c-4060-8702-9727e50cd65a.png">


### Expected result AFTER applying this Pull Request
<img width="448" alt="grafik" src="https://user-images.githubusercontent.com/828371/166142311-095d3b88-f9d7-479f-881a-62fa881d0aa6.png">


### Documentation Changes Required
No
